client/lib/plans/constants.js

```diff
@@ -518,6 +519,17 @@ export const FEATURES_LIST = {
 		)
 	},
 
+	[ FEATURE_6GB_STORAGE ]: {
+		getSlug: () => FEATURE_6GB_STORAGE,
+		getTitle: () => i18n.translate( '{{strong}}6GB{{/strong}} Storage Space', {
+			components: {
+				strong: <strong />
+			}
+		} ),		getDescription: () => i18n.translate(
+			'Storage space for adding images and documents to your website.'
```

Comment:
Let's copy what the other space upgrades use for the description:

`With increased storage space you'll be able to upload more images, videos, audio, and documents to your website.`